setup.py

```diff
@@ -1,4 +1,6 @@
-# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Copyright 2024 The Emukit Authors. All Rights Reserved.
+#
+# Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
There are 3 files or so that had different headers with an extended license mention (It may not be visible in the diff). This file (`setup.py`) is one of them. [Here](https://github.com/EmuKit/emukit/blob/main/setup.py) is the unchanged file for comparison. Can you have a look at the original and updated version and comment on how you think we should handle these `special' files?

Reply:
I'd say let's make all headers look the same, so that there are no special files going forward

Reply:
OK, I'll change those to match the other headers. 

I just saw that some of the non-py files have no headers (e.g., `pyproject.toml`) shall I add headers to those as well? Also can you have a look at the `NOTICE` file? I just added one line there.